cli/cmd/templates/source/resources/table.go.tpl

```diff
@@ -0,0 +1,23 @@
+package resources
+
+import (
+	"github.com/{{.Org}}/cq-source-{{.Name}}/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func SampleTable() *schema.Table {
+	return &schema.Table{
+		Name:     "{{.Name}}_sample_table",
+		Resolver: fetchSampleTable,
+		Columns: []schema.Column{
+      {
+        Name: "column",
+        Type: schema.String,
+      },
+    },
+  }
+}
+
+func fetchSampleTable(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- interface{}) error {
```

Comment:
I think we should re-enforce the convention of placing fetch functions inside separate `_fetch.go` files here